DEVELOPER_GUIDE.md

```diff
@@ -93,8 +97,16 @@ snapshots/
                 ├── maven-metadata.xml.sha1
                 ├── maven-metadata.xml.sha256
                 └── maven-metadata.xml.sha512
-
-
 ```
+
 1. Change the url from ``"https://aws.oss.sonatype.org/content/repositories/snapshots"`` to your local path and comment out the credentials under publishing/repositories in build.gradle.
 2. Run ```./gradlew publishPluginZipPublicationToSnapshotsRepository```.
+
+### Adding Workflow Steps
+
+To add functionality to workflows, add new Workflow Steps to the [`org.opensearch.flowframework.workflow`](https://github.com/opensearch-project/flow-framework/tree/main/src/main/java/org/opensearch/flowframework/workflow) package.
+1. Implement the [Workflow](https://github.com/opensearch-project/flow-framework/blob/main/src/main/java/org/opensearch/flowframework/workflow/WorkflowStep.java) interface. See existing steps for examples for input, output, and API execution.
```

Comment:
Should we add a link to an existing example like https://github.com/opensearch-project/flow-framework/blob/main/src/main/java/org/opensearch/flowframework/workflow/CreateConnectorStep.java so they know what to reference?

Reply:
I linked to the package folder with lots of examples so I think that's probably good enough.